### ISSUE TITLE
Suggestion: Add a debug varible in the object Page

### ISSUE THREAD

**Issue by xspager** [sailorproject/sailor#116]
Might be useful to have a debug variable on the object Page so the template could decide to include or not a block of script like Google Analytic for example.

**Comment by Etiene** [comment]
you mean something like `page:inspect`? https://github.com/sailorproject/sailor/blob/master/docs/manual_page_object.md#pageinspect-var-

**Comment by xspager** [comment]
No, just value true or false I can use on a template to decide to include or not on the html the google analytics script.

**Comment by Etiene** [comment]
I see, you could add it at `conf/conf`maybe. We already have a `debug.inspect` field. As it's we only use it for inspect, maybe we should replace this for a simpler thing? Remove inspect and have debug only?

Then at the views you can 
`local debug = require"conf.conf".debug

**Comment by xspager** [comment]
Would this be accessible on the view?

**Comment by Etiene** [comment]
Yes

2016-04-20 17:53 GMT+02:00 Daniel Lemos <notifications@github.com>:

> Would this be accessible on the view?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/sailorproject/sailor/issues/116#issuecomment-212487247>
>



-- 
Etiene Dalcol

Software Engineering student at ENSTA Bretagne and PUC-Rio
Sailor Developer http://sailorproject.org
Lua Ladies Founder http://lualadies.org

**Comment by xspager** [comment]
Oh, sorry, let me try again... Yes, `conf.lua` is the right place to this value. `debug.inspect` is very useful this would be something extra.
